staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3390,6 +3390,14 @@ type TopologySpreadConstraint struct {
 	// in their corresponding topology domain.
 	// +optional
 	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty" protobuf:"bytes,4,opt,name=labelSelector"`
+	// MinDomains indicates a minimum number of domains.
+	// When the number of domains with matching topology keys is less than `minDomains`,
+	// Pod Topology Spread treats "global minimum" as 0.
+	// As a result, when the number of domains is less than `minDomains`,
+	// scheduler doesn't schedule a matching Pod to Nodes on the domains that have the same or more number of matching Pods as `maxSkew`.
+	// Default value is 0. When value is different than 0, WhenUnsatisfiable must be DoNotSchedule.
+	// +optional
+	MinDomains *int32
```

Comment:
you are missing the json tag. The protobuff one can be autogenerated running `make update`.

Reply:
Oops. Thanks.